### DISCUSSION TITLE
v0.26 release doesn't have a search box anymore

### DISCUSSION THREAD

**Discussion by huyz** [Anze/KeyCluCask#71] [Q&A] [answered]
Is that on purposes or a bug?

Before:
![screenshot 2024-04-26T084200Z@2x](https://github.com/Anze/KeyCluCask/assets/128394/c9d528ab-3693-4462-9020-4de1d51b0900)

V0.26:
![screenshot 2024-04-26T084338Z@2x](https://github.com/Anze/KeyCluCask/assets/128394/82db31b9-a4ce-490b-8665-78834d9c4e1b)

**Comment by Anze** [comment, marked as answer]
Hey [huyz](https://github.com/huyz),

In the release notes, I mentioned that the filtering shortcuts functionality has been relocated to "Experimental Features" due to its unstable state. This move was made because it might not deliver the best user experience when used. To access this feature, you'll need to [follow the instructions](https://github.com/Anze/KeyCluCask/wiki/Experimental-Features) provided to enable `Experimental Features`.
